### PR TITLE
Fixing processing of left/right arrows in TextBox

### LINKED ISSUE #1444: TextBox doesn't handle left/right arrows where it used to
`ChatOverlay` is an example of this occurring

### PR DESCRIPTION
**Pull request by pdrapoport**
Processing of Left/Right keys was disabled in FocusedTextBox. Setting value of "HandleLeftRightArrows" to true leads to expected behaviour.
If approved, this PR will fix [osu-framework issue 1444](https://github.com/ppy/osu-framework/issues/1444)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ This is still heavily under development and is not intended for end-user use. Th
 
 We welcome all contributions, but keep in mind that we already have a lot of the UI designed. If you wish to work on something with the intention on having it included in the official distribution, please open an issue for discussion and we will give you what you need from a design perspective to proceed. If you want to make *changes* to the design, we recommend you open an issue with your intentions before spending too much time, to ensure no effort is wasted.
 
-Please make sure you are familiar with the [development and testing](https://github.com/ppy/osu-framework/wiki/Development-and-Testing) procedure we have set up. New component development, and where possible, bug fixing and debugging existing components **should always be done under VisualTests**.
+Please make sure you are familiar with the [development and testing](https://github.com/ppy/osu-framework/wiki/Development-and-Testing) procedure we have set up. New component development, and where possible, bug fixing and debugging existing components **should always be done under VisualTests for Visual Studio**.
```
Comment (jorolf) [inline README.md]: ???

Comment (pdrapoport) [reply README.md]: Sorry. Lack of team cohesion. This changes were reverted.

**Comment by peppy** [comment]
I'm going to close this and fix in a separate PR. Your commits are weird.

**Comment by pdrapoport** [comment]
Can you be more specific?

**Comment by peppy** [comment]
Check the commits tab https://github.com/ppy/osu/pull/2173/commits you have changed/merged random files.

**Comment by peppy** [comment]
Please learn how to use `git rebase` or `git reset --mixed` at least.

**Comment by pdrapoport** [comment]
Ok, thanks for feedback.

**Pull request closed without merging**